datm/datm_datamode_clmncep_mod.F90

```diff
@@ -35,8 +35,6 @@ module datm_datamode_clmncep_mod
   real(r8), pointer :: Sa_dens(:)           => null()
   real(r8), pointer :: Sa_pbot(:)           => null()
   real(r8), pointer :: Sa_pslv(:)           => null()
-  real(r8), pointer :: Sa_co2prog(:)        => null() ! co2
-  real(r8), pointer :: Sa_co2diag(:)        => null() ! co2
```

Comment:
Why have you removed these? I think there are modes that are needed for clm that are not in these tests.

Reply:
These variables are not used anywhere in this file.  They can always be replaced if they are used at some future date.